### PR TITLE
Updates support

### PR DESCRIPTION
**Pull request by JerryNixon**
Updated Support.md for GA

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+:+1: - upvote
 
-We recommend that you search through the existing issues to see if there is a similar issue that you can comment on. If you find an existing issue, please add a comment to the existing issue instead of creating a new one.
+:-1: - downvote
 
-When an issue is raised, a member of the team will respond as soon as possible, so please be patient.
+### Azure services support tickets
+If your solution hosts Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request). 
+
+### Azure database support tickets
```
Comment (Aniruddh25) [inline SUPPORT.md:27]:
```suggestion
### Azure Database support tickets
```

File: SUPPORT.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
+
+If your solution uses an Azure database, including Azure SQL, Azure Cosmos DB, Azure Database for PostgreSQL, Azure Database for MySQL, and Azure SQL Data Warehouse, then the cause of your issue could be with that database. If you wish to open a support ticket with that database, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).
+
+Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).
```
Comment (Aniruddh25) [inline SUPPORT.md:31]: how is this paragraph 

> Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).

different than the content above `Azure services support tickets`

File: SUPPORT.md [hunk 3]
```diff
@@ -1,18 +1,34 @@
 # Support
 
+Data API builder is an open-source project with contributions from Microsoft and the community to build a configuration-based engine in a Docker-friendly container that easily adds a secure and feature-rich Data API over databases. Data API builder is not a Microsoft product and, as a result, is not supported by Microsoft as a product would be.
+
 ## How to file issues and get help
 
-### Azure support tickets
+### GitHub Issues
+
+Engineers who contribute to and work on Data API builder periodically monitor and review issues added to [the repository's issues list](https://github.com/Azure/data-api-builder/issues). The period of review is not set, unlike service level agreements (SLA) with Microsoft products, and resolution is not guaranteed. When an issue is raised, a member of the team will respond as soon as possible; please be patient.
```
Comment (seantleonard) [inline SUPPORT.md:9]:
```suggestion
Engineers who contribute to and work on Data API builder periodically monitor and review issues added to [the repository's issues list](https://github.com/Azure/data-api-builder/issues). The period of review is not set by a service level agreement (SLA), and resolution is not guaranteed. When an issue is raised, a member of the team will respond as soon as possible; please be patient.
```

File: SUPPORT.md [hunk 4]
```diff
@@ -1,18 +1,34 @@
...
+
+Repository issues track bugs, questions, and feature requests. Creating issues requires a GitHub account; templates help ensure information is used for investigation. Repository issues may be added to the backlog by engineers and become triaged in our future roadmap.
```
Comment (seantleonard) [inline SUPPORT.md:11]:
```suggestion
Repository issues track bugs, questions, and feature requests. Creating issues requires a GitHub account; issue templates prompt for details that will help engineers reproduce your issue and more effectively complete an investigation. Repository issues may be added to the backlog by engineers and become triaged in our future roadmap.
```

File: SUPPORT.md (hunk 2 above)
Comment (JerryNixon) [reply SUPPORT.md:31]: It isn't a different link, just a different context to get there. I think this is okay.

File: SUPPORT.md [hunk 5]
```diff
@@ -1,18 +1,34 @@
...
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.
 
-We use [GitHub Issues](https://github.com/azure/data-api-builder/issues) to track bugs, questions, and feature requests. Opening GitHub issues requires you to have a free to create GitHub account. There is a GitHub Issues template to help you provide the information we need to investigate your issue.
+:+1: - upvote
 
-We recommend that you search through the existing issues to see if there is a similar issue that you can comment on. If you find an existing issue, please add a comment to the existing issue instead of creating a new one.
+:-1: - downvote
 
-When an issue is raised, a member of the team will respond as soon as possible, so please be patient.
+### Azure services support tickets
```
Comment (Aniruddh25) [inline SUPPORT.md:24]:
```suggestion
### Azure services support tickets


```

nit: leave a line after subtitle like others.

**Review by Aniruddh25 — APPROVED** [review]
LGTM. Thank you for these clarifications.

**Review by aaronburtle — APPROVED** [review]
Thanks Jerry, this looks great!

**Pull request merged**